manifests/rule.pp

```diff
@@ -64,48 +67,59 @@
 #   If undefined it defaults to the setting in logrotate
 # @param size
 # @param sharedscripts
+# @param shred
```

Comment:
why not add noshred, as well?

Reply:
Other params in the template that look like `feature` and `nofeature` set `feature` if the bool is true and set `nofeature` if the bool is false. I was hoping to make the behavior consistent.